app/utils/graph-tools.js

```diff
@@ -264,6 +268,61 @@ const positionStage = (x, y, stage) => {
   }
 };
 
+const calcNodeDepth = graph => {
```

Comment:
```suggestion
const calcNodeDepths = graph => {
```
Renaming this to be a plural better indicates that all node depths are returned in the object